registry/default/magicui/script-copy-btn.tsx

```diff
@@ -0,0 +1,140 @@
+"use client";
+
+import { useState, useEffect } from "react";
+import { Button } from "@/components/ui/button";
+import { Check, Copy } from "lucide-react";
+import { codeToHtml } from "shiki";
+import { motion } from "framer-motion";
+import { useTheme } from "next-themes";
```

Comment:
what should be done for non next.js users?

Reply:
It will work for non-next.js users too.